src/features/MentorPage/MentorCard/MentorCard.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
+import { handleSetVisibleCardProps } from '../MentorPage';
+import MentorCardHeader from './MentorCardHeader';
+import CloseIcon from '../../../static/icons/icon-close.svg';
+import MentorContent from './MentorContent';
+
+/**
+ * Selected mentor card in mentor page
+ */
+
+type Props = {
+  setVisibleCard: ({
+    shouldShowMentorCard,
+    mentorCardData,
+  }: handleSetVisibleCardProps) => void;
+  mentorCardData: ListCardProps;
+};
+
+const MentorCard = ({ setVisibleCard, mentorCardData }: Props) => {
+  const handleClick = () => {
+    const shouldShowMentorCard = false;
+    setVisibleCard({ shouldShowMentorCard, mentorCardData });
+  };
+
+  if (mentorCardData) {
+    return (
+      <MentorCardContainer>
+        <StyledMentorCard className="mentorikortti">
+          <MentorCardHeader mentorCardData={mentorCardData} />
+          <CloseButton onClick={handleClick}></CloseButton>
+          <MentorContent mentorCardData={mentorCardData} />
+        </StyledMentorCard>
+      </MentorCardContainer>
+    );
+  } else {
+    return <></>;
+  }
+};
+
+const CloseButton = styled.button`
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  background-image: url(${CloseIcon});
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 2rem;
+  width: 2rem;
+  background-color: transparent;
+  border: none;
+  appearance: none;
+  z-index: 10;
+`;
```

Comment:
Last thing: Instead of defining lot of different buttons in different components, I would rather see a Button component which is shared across many components. 
We could have two kinds of buttons => IconButtons, and TextButtons.
Both Button-types should receive variants as Props similarly like our new Text-component. This way it is easy to just have same looking and functioning buttons accross the application.

Reply:
How very reasonable of you :D